doc/backend-development.md

```diff
@@ -7,17 +7,19 @@ In order to keep things tidy, we have agreed on some conventions:
 2. Name this namespace `datahike-{backendname}.core`.
 3. Make sure, all multimethods are defined for the new backend you are developing.
 
-As an example, you may have a look at the implementation of our [Postgres](https://www.postgresql.org/) backend, i.e.
-[datahike-postgres](https://github.com/replikativ/datahike-postgres).
+As an example, you may have a look at the implementation of our [JDBC](https://docs.oracle.com/javase/8/docs/technotes/guides/jdbc/) backend, i.e.
+[datahike-jdbc](https://github.com/replikativ/datahike-jdbc).
 
 ##  Template
 
-[Here, we provide a basic template for a backend implementation](https://github.com/replikativ/datahike-backend-template). Following are bracketed text pieces defining placeholder values, you should replace as follows:
+We provide a basic template for a backend implementation [here](https://github.com/replikativ/datahike-backend-template). 
+
+Following are bracketed text pieces defining placeholder values, you should replace as follows:
 - **backendname** surprisingly should be the name of your backend.
 - **backendID** should be a `keyword` to identify your backend on request. At this moment, datahike ships with backends identified by `:mem` and `:file`, so do not use those.
 - **indexID** should be a `keyword` identifying an index to be used as default for your backend. So far, you can choose between the following: 
   - `:datahike.index/hitchhiker-tree` 
-  - `:datahike.index/persistent-set`
+  - `:datahike.index/persistent-set` -> I think so far it would not work for the persistent set or would it?
```

Comment:
Right, this should be removed then for now and added to the roadmap.

Reply:
Done

Reply:
Only check, that it is in the right spot on the roadmap

Reply:
Pushed it to version 0.5.0 now